### PR TITLE
Add release notes for 0.25.3

### PR DESCRIPTION
**Pull request by bioball**

### CODE REVIEW

File: docs/modules/release-notes/pages/changelog.adoc [hunk 1]
```diff
@@ -1,6 +1,38 @@
...
+=== Fixes
+
+* Fixes some issues with generated pkldoc websites (link:https://github.com/apple/pkl/pull/357[#357], link:https://github.com/apple/pkl/pull/362[#362])
+* Fixes a bug where amending a module that defines an abstract class might cause a Java `AssertionError` (link:https://github.com/apple/pkl/pull/319[#319])
+* Fixes a bug where a for/when generator within a lambda declared with `new {}` syntax might not resolve variables correctly (link:https://github.com/apple/pkl/pull/297[#297])
+* Fixes a bug where `const` and `local` modifiers are not exported when obtaining a class's mirror in `pkl:reflect` (link:https://github.com/apple/pkl/pull/265[#265]).
+
+=== Changes
+
+* Add
```
Comment (holzensp) [inline docs/modules/release-notes/pages/changelog.adoc:16]: Nothing in it?

Comment (bioball) [inline docs/modules/release-notes/pages/changelog.adoc:16]:
```suggestion
```

**Comment by StefMa** [comment]
What about mentioning @translatenix? 😇
He actually did quite a lot 🙃

**Comment by bioball** [comment]
He certainly did, although those commits are going to land in the next release (0.26) rather than this release!

**Review by stackoverflow — APPROVED** [review]

**Pull request merged**